site/en/docs/extensions/mv3/user_interface/index.md

```diff
@@ -35,51 +60,50 @@ The `"action"` field is registered in the manifest.
 }
 ```
 
-Declaring `"action"` keeps the icon colorized, indicating the extension is available to
-users.
+###  Activate the action conditionally  {: #activate_pages }
```

Comment:
Not required for this PR, but we should probably talk about enabled/disabled action behaviors before diving into programmatically or declaratively modifying this state (see also ["Enabled state"](https://deploy-preview-2384--developer-chrome-com.netlify.app/docs/extensions/reference/action/#enabled-state)). Also, we don't currently cover that the [enable()](https://developer.chrome.com/docs/extensions/reference/action/#method-enable) and [disable()](https://developer.chrome.com/docs/extensions/reference/action/#method-disable) methods on this page.